### PR TITLE
glass: add unique service name validation

### PR DESCRIPTION
**Pull request by s0nea**
## Checklist
- [ ] References issues, create if required
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins test tumbleweed`
- `jenkins run tumbleweed`
- `jenkins test leap`
- `jenkins run leap`

</details>

### CODE REVIEW

File: src/glass/src/app/core/modals/cephfs/cephfs-modal.component.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ import {
 } from '~/app/shared/services/api/services.service';
 import { NotificationService } from '~/app/shared/services/notification.service';
 
+import { GlassValidators } from '../../../shared/forms/validators';
```
Comment (votdev) [inline src/glass/src/app/core/modals/cephfs/cephfs-modal.component.ts:33]: Please use the absolute path here.

Comment (s0nea) [reply src/glass/src/app/core/modals/cephfs/cephfs-modal.component.ts:33]: Hmpf... why is this happening again... Thanks!

File: src/glass/src/app/shared/services/api/services.service.ts [hunk 2]
```diff
@@ -116,6 +131,16 @@ export class ServicesService {
     return this.http.get<ServiceDesc[]>(`${this.url}/`);
   }
 
+  public validateServiceName(service_name: string): Observable<boolean> {
```
Comment (votdev) [inline src/glass/src/app/shared/services/api/services.service.ts:134]: What about using this name? Looks clearer to me.
```suggestion
  public exists(service_name: string): Observable<boolean> {
```

**Review by votdev — CHANGES_REQUESTED** [review]

File: src/glass/src/app/shared/services/api/services.service.ts (hunk 2 above)
Comment (s0nea) [reply src/glass/src/app/shared/services/api/services.service.ts:134]: Yes, fine with me

**Review by jecluis — APPROVED** [review]

**Review by votdev — APPROVED** [review]

**Pull request merged**